sherpa/astro/data.py

```diff
@@ -18,8 +18,74 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
-"""
-Classes for storing, inspecting, and manipulating astronomical data sets
+"""Classes for storing, inspecting, and manipulating astronomical data sets.
+
+The two types of Astronomical data supported in this module are
+two-dimensional images (:py:class:`DataIMG`) and X-ray spectra
+(:py:class:`DataPHA`), along with associated response information
+(:py:class:`DataARF` and :py:class:`DataRMF`). These objects can be
+constructed directly or read from :term:`FITS` files using the
+:py:mod:`sherpa.astro.io` routines.
+
+Both types of data extend the capabilities of the
+:py:class:`sherpa.data.Data` class:
+
+- using geometric shapes (regions) to filter images;
+
+- support different units for filtering images (logical, physical, and
+  :term:`WCS`), depending on the available metadata;
+
+- support different analysis units for filtering and display for
+  :term:`PHA` files (channels, energy, and wavelengths);
+
+- dynamically re-bin PHA data to improve the signal to noise (grouping
+  and quality);
+
+- and automatically support one or more spectra that define the
+  background for the observation (for PHA files) that can then be
+  subtracted from the data or a background model fit to them.
+
+Notes
+-----
+
+Some functionality depends on the presence of the region and grouping
+Sherpa modules, which are optional components of Sherpa.
+
+Notebook support
+----------------
+
+The Data objects support for Jupyter notebooks, and will display a
```

Comment:
grammatically, should  "The Data objects support for Jupyter notebooks"   be  something like  "The Data objects support Jupyter notebooks" (without the 'for') or something like 'The Data objects provide/offer support for Jupyter notebooks'.

Reply:
Well, my original text is poor!

Reply:
I've changed this slightly. Hopefully more readable.